docs/api.md

```diff
@@ -2,21 +2,32 @@
 
 #### Create a new infbox template builder instance
 All parameters  for `InfoboxTemplateBuilder` are optional, though in order to save to MediaWiki, a title value is required inside the `persistOptions` object. 
+
 ```javascript
-var infobox = new InfoboxTemplateBuilder({
+var adapter = {
+  name: 'XMLSerializer',
```

Comment:
That's for registering/creating infoboxes. This is just for specifying which built-in ones run.